.github/workflows/benchmarks-reusable.yml

```diff
@@ -176,6 +176,16 @@ jobs:
         -B${{github.workspace}}/umf_build
         -DUMF_BUILD_BENCHMARKS=ON
         -DUMF_TESTS_FAIL_ON_SKIP=ON
```

Comment:
meant to add it here:

FYI, you don't need `UMF_TESTS_FAIL_ON_SKIP=ON` if you disabled tests 😉